src/include/daos/drpc_modules.h

```diff
@@ -77,6 +77,7 @@ enum drpc_mgmt_method {
 	DRPC_METHOD_MGMT_REINTEGRATE		= 225,
 	DRPC_METHOD_MGMT_CONT_SET_OWNER		= 226,
 	DRPC_METHOD_MGMT_EXCLUDE		= 227,
+	DRPC_METHOD_MGMT_POOL_EVICT		= 229,
```

Comment:
what is 228? what's the reason for skipping from 227->229?

Reply:
https://github.com/daos-stack/daos/pull/2768

My PR for dmg extend used 228. I thought that might land first so I just skipped 228 in the PR so I wouldn't need to go back and change this one.

If it's preferable I can change this to 228 and the rebase and fix whichever one lands second?